.github/dependabot.yml

```diff
@@ -8,4 +8,7 @@ updates:
   - package-ecosystem: "npm" # See documentation for possible values
     directory: "/" # Location of package manifests
     schedule:
-      interval: "daily"
\ No newline at end of file
+      interval: "daily"
+    ignore:
+      - dependency-name: "react-native"
+      - dependency-name: "@react-native/metro-config"
```

Comment:
added dependabot configuration to ignore react-native and @react-native/metro-config dependencies